addOns/network/src/main/resources/org/zaproxy/addon/network/resources/Messages.properties

```diff
@@ -379,6 +379,7 @@ network.ui.options.servercertificates.button.save = Save
 network.ui.options.servercertificates.button.view = View
 
 network.ui.options.servercertificates.field.certvalidity = Validity in Days:
+network.ui.options.servercertificates.field.crldistpoint = CRL DP URL:
```

Comment:
Certificate Revocation List Distribution Point URL:

Reply:
That will not fit well in the UI.

Reply:
Then how about:

> CRL Distribution Point URL:

I guess we can assume people will get CRL. But 'DP' is a bit of a stretch.

Reply:
If it still needs to be shorter, drop the word 'Point'?

Reply:
The 2nd suggestion should be fine.

Reply:
I would drop the 'URL', which is implied, and shorter the label to 'CRL Dist. Point' ? I find it strange without the word 'Point'

Reply:
I'm fine with that.